CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# dbt_pinterest v1.0.1
+
+[PR #49](https://github.com/fivetran/dbt_pinterest/pull/49) includes the following updates:
+
+## Bugfix
+- Added configuration option to conditionally enable the `keyword_report` source based on the `ad_reporting__pinterest_ads_enabled` and `pinterest__using_keywords` variables. This allows for better control when the keyword report source is disabled in your project.
```

Comment:
Request to update the bugfix entry to also include the `keyword_history` as the update was also applied to that source.